### DISCUSSION TITLE
Saw the latest release

### DISCUSSION THREAD

**Discussion by Vilhelm-Ian** [mortii/anki-morphs#299] [General]
Thank you very much for how long you worked on the addon. I understand your decision not to want to work on the addon further

**Comment by mortii** [comment]
It's bittersweet, but everything comes to an end at some point. Thank you for all your contributions, it's been much appreciated :)

**Reply by mortii** [reply]
@aleksejrs 

bitter: I couldn't make AnkiMorphs reach its full potential
sweet: I no longer have to sacrifice my time and effort into developing anki add-ons

so no, i won't contribute to frequencyman, nor anything anki related. I hope those project thrive though, the world is a better place because of them.

Have a good one! ✌️

**Comment by xofm31** [comment]
@mortii Thank you for making this beautiful addon. Do you know when they will be making this transition to Svelte? I will probably not upgrade Anki if it isn't compatible with AnkiMorphs...

**Reply by Gardengul** [reply]
You can always downgrade the anki version when you find that the addon does not work.

Maybe it will help to have a second version of anki on the device
[Running from a Flash Drive](https://docs.ankiweb.net/files.html#running-from-a-flash-drive)

**Reply by mortii** [reply]
> Do you know when they will be making this transition to Svelte?

There isn't a date yet as far as I know, but [Damien mentioned](https://forums.ankiweb.net/t/revisiting-onboarding-decks-and-tutorials/43692/89) a possible feature freeze to speed up the transition.

> Thank you for making this beautiful addon.

Thank you @xofm31, it's been a pleasure working with you :)

**Comment by Raytracer42** [comment]
Thank you for making AnkiMorps! And basically enabling the use of morphman-like features with fsrs anki versions. I think you helped a lot of people. 

I hope anki developers would maintain compatability of "old" addons as long as possible. I imagine your addon is not the only one that would need to be completely rewritten.

**Comment by RobHelgeson** [comment]
Hey Mortii!

I just wanted to get your thoughts on the whole Svelte thing. 

I’m a full stack dev, leaning a bit more towards front end than back end. I have some interest in learning Svelte, and was wondering what your thoughts on a couple of options were.

1. Would you consider keeping on the project if I undertook conversion of the Qt stuff to Svelte?
2. If not, would you be opposed to me creating a fork and doing it myself?

I don’t know if I REALLY want to undertake it, but before I started to give it serious thought, I wanted to have a chat. 

I’d love to hear any and all your thoughts.

**Reply by mortii** [reply]
> 1. Would you consider keeping on the project if I undertook conversion of the Qt stuff to Svelte?

Thanks for offering! I would definitely be willing to "maintain" the project in the sense of bug fixing and reviewing pull requests. However, I'm no longer passionate enough to develop new features or undertake major rewrites.

> 2. If not, would you be opposed to me creating a fork and doing it myself?

If I ever decline a pull request or become unresponsive, then absolutely start publishing your own fork if you want to, I have no problem with that.

> I don’t know if I REALLY want to undertake it, but before I started to give it serious thought

If you decide to give it a shot, maybe start with the `Known Morphs Exporter` and see how that goes? No pressure though, bail early if you're not feeling it.
